mindsdb/api/http/initialize.py

```diff
@@ -302,7 +302,7 @@ def before_request():
             and check_auth() is False
         ):
             return http_error(
-                403, 'Forbidden',
+                HTTPStatus.UNAUTHORIZED, 'Forbidden',
```

Comment:
if status is UNAUTHORIZED maybe message should be too? (not 'Forbidden')

Reply:
UNAUTHORIZED 401 seems more appropriate here, frontend is notified